### ISSUE TITLE
Fingerprints not added to the manifests

### ISSUE THREAD

**Issue by Turbo87** [davewasmer/ember-cli-favicon#78]
When I build my app with `ember build -prod` it generates all the favicon assets, then fingerprints them correctly, but then it does not update the generated manifest files with the new filenames. Ultimately the manifests now point to files that don't exist, while the fingerprinted files are not referenced anywhere either.

My temporary workaround is:
```js
    fingerprint: {
      exclude: ['android-chrome', 'coast', 'firefox_app', 'yandex-browser'],
    },
```

**Comment by Mike-Neto** [comment]
Thanks, I also noticed this issue and added your workaround to my base ember-cli-build config.

**Comment by Exelord** [comment]
Solved in #82

**Comment by jlami** [comment]
I originally solved this problem by including `xml` en `json` as extra extensions that the fingerprint module will search for content to replace:
```
fingerprint: {
      replaceExtensions: ['html', 'css', 'js', 'xml', 'json'],
    },
```
I don't know if I'm really happy with completely disabling fingerprinting. Would it not be better to find a way to include manifest.json and other files generated by this plugin in the replace files?

Forcing the parent.options.fingerprint.replaceExtensions to include `json` en `xml` feels wrong. But broccoli-asset-rev does not seem to have a way to explicitly say which files will contain references to fingerprinted files other than by extension.
